subprojects/build-operations/build-operations.gradle.kts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id("gradlebuild.distribution.implementation-java")
+    id("gradlebuild.publish-public-libraries")
+}
+
+description = "Build operations are our way to inspect the process of executing a build"
+
+gradlebuildJava.usedInWorkers()
+
+dependencies {
+    implementation(project(":baseAnnotations"))
+    implementation(libs.guava)
```

Comment:
Can we get rid of the `guava` dependency`?

Reply:
I removed the Guava dependency, and also removed a use of a Java 8 API. Java 6 compatibility should be airtight now.

Reply:
Nice!